Makefile

```diff
@@ -32,5 +32,6 @@ test.e2e:
 
 .PHONY: test.integration
 test.integration:
+	echo ${KONG_ENTERPRISE_LICENSE}
```

Comment:
don't echo secrets, as this will log them in CI and they will need to be invalidated. I see that this hasn't actually worked yet, so just remove this before doing anything else. Also I believe Github actions will censor secrets anyway (e.g. `*****` will be used in place of it).

Reply:
atm, this is echo the environment variable value. and it is empty. pure debug purpose.

Reply:
Right I get that right now it's not even working, what I'm saying is that we _fundamentally_ should never be trying to echo secrets in a place that CI would log them. Looks like it's removed now though, and we didn't end up emitting the secret to logs from what I can see so this is all set.